### ISSUE TITLE
Some screens cause errors.

### ISSUE THREAD

**Issue by KaosDragon** [IIInitiationnn/BloodEmporium#26]
Benedict's Quote at level 2 caused this one. 
[debug-23-02-27 21-59-44.log](https://github.com/IIInitiationnn/BloodEmporium/files/10846314/debug-23-02-27.21-59-44.log)

Hitting prestige 1 caused this. 
[debug-23-02-27 22-34-20.log](https://github.com/IIInitiationnn/BloodEmporium/files/10846315/debug-23-02-27.22-34-20.log)

I hope these logs help.

**Comment by IIInitiationnn** [comment]
Hey! I believe this has been resolved with the new 1.0.0 update. I will mark this as fixed for now, but if you run into the same issues on prestige / new level screens, please feel free to reopen this thread.

**Comment by KaosDragon** [comment]
They only ever happened the one time I made for this issue report. I've prestiged/hit level 2 on other characters since then and didn't have any problem. My only guess on why is for the level 2 one, I had left it set on survivor while leveling up Myers. I fixed that afterwards and the prestige issue happened when Mike hit 50 while set up properly.

In hindsight I probably should've mentioned this stuff when I made the issue...
